core/src/main/resources/hudson/PluginManager/table.properties

```diff
@@ -34,6 +37,10 @@ depCoreWarning=\
  Warning: This plugin requires dependent plugins that require Jenkins {0} or newer. \
  Jenkins will refuse to load the dependent plugins requiring a newer version of Jenkins, \
  and in turn loading this plugin will fail.
+depJavaWarning=\
+ Warning: This plugin requires dependent plugins that require Java {0} or newer. \
```

Comment:
_requires dependent plugins_ reads a little awkwardly—it is redundant. I might suggest something like

> Warning: this plugin requires other plugins which in turn require Java…